Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM debian:stretch
+
+## Install min deps
+RUN apt-get update
+
+## Install NAPALM & underlying libraries dependencies
+RUN apt-get install -y python-cffi python-dev libxslt1-dev libssl-dev libffi-dev \
+    && apt-get install -y python-pip \
+    && pip install -U cffi \
+    && pip install -U cryptography \
+    && pip install napalm
```

Comment:
Change this to `pip install .` ?  Do we want the Dockerfile to install the latest release, or install the latest dev?

Reply:
`pip install .` might be a good idea indeed

Reply:
Updated - see 723fd72